src/submission/models.py

```diff
@@ -322,14 +322,21 @@ class Meta:
         null=True,
         help_text='Funder DOI (optional). Enter as a full Uniform '
                   'Resource Identifier (URI), such as '
-                  'http://dx.doi.org/10.13039/501100021082',
+                  'https://dx.doi.org/10.13039/501100021082',
     )
     funding_id = models.CharField(
         max_length=500,
         blank=True,
         null=True,
         help_text="The grant ID (optional). Enter the ID by itself",
     )
+    funding_statement = models.TextField(
+        blank=True,
+        help_text=_("Additional information regarding this funding entry")
+    )
```

Comment:
Funders have a m2m relationship with articles. Shouldn't this exist on the linking table, rather than here? Otherwise I can see two different articles competing for a Funder record and diverging funding statements competing against each other